core/src/main/java/org/jdbi/v3/core/statement/Binding.java

```diff
@@ -151,19 +151,75 @@ public Optional<Argument> findForPosition(int position) {
 
     @Override
     public String toString() {
-        String positionalsDescription = positionals.entrySet().stream()
-            .map(x -> x.getKey().toString() + ':' + unwrap(x.getValue()))
-            .collect(Collectors.joining(","));
-
-        String namedDescription = named.entrySet().stream()
-            .map(x -> x.getKey() + ':' + unwrap(x.getValue()))
-            .collect(Collectors.joining(","));
-
-        String found = namedArgumentFinder.stream()
-            .map(Object::toString)
-            .collect(Collectors.joining(","));
+        return toString(Integer.MAX_VALUE);
+    }
 
-        return "{positional:{" + positionalsDescription + "}, named:{" + namedDescription + "}, finder:[" + found + "]}";
+    public String toString(final int maxLength) {
```

Comment:
it may make sense to name this different and we may be able to make that package private.

Reply:
Unfortunately it is used by the `JdbiOpenTelemetryPlugin` which is not in the `statement` package, so it must be public (or otel support moves to `.statement` but that feels pretty weird)

Reply:
can we call it something not `toString(int len)`? e.g. `formatOutput(int len)` or so? `toString` is a very loaded name in java land and it may confuse people to have two methods with that name.